app/src/main/java/com/example/studentarena/utilities/GetRelativeTime.java

```diff
@@ -0,0 +1,47 @@
+package com.example.studentarena.utilities;
+
+import android.content.Context;
+import android.text.format.DateUtils;
+import android.widget.ImageView;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+public class GetRelativeTime {
+
+    private static final int SECOND_MILLIS = 1000;
+    private static final int MINUTE_MILLIS = 60 * SECOND_MILLIS;
+    private static final int HOUR_MILLIS = 60 * MINUTE_MILLIS;
+    private static final int DAY_MILLIS = 24 * HOUR_MILLIS;
+
+    public static String getSimpleTime(Date inputDate) {
+        String twitterFormat = "EEE MMM dd HH:mm:ss ZZZZZ yyyy";
+        SimpleDateFormat sf = new SimpleDateFormat(twitterFormat, Locale.ENGLISH);
+        sf.setLenient(true);
+
+        long time = inputDate.getTime();
+        long now = System.currentTimeMillis();
```

Comment:
suggested GitHub issue to follow up on later: figure out how to handle timezones

let's say you're in Washington and i'm in Michigan and we both create posts at 4:00pm. i'm three hours ahead so my post is way older, but if all parse has to go on is "4pm" it'll think we created them at the same time. to handle that, many computer things covert times to UTC (Coordinated Universal Time) before saving them. UTC is basically a special timezone that doesn't change according to Daylight Savings Time or anything, and then they convert back to a local timezone before displaying the time to the user

i think Parse automatically creates its `created_at` and `updated_at` timestamps using UTC already, but your `System.currentTimeMillis()` will get a timestamp that uses your system timezone. you probably want to convert one to the other before comparing to see how much time passed between them (see [stackoverflow](https://stackoverflow.com/questions/41062341/time-conversion-milliseconds-local-time-to-milliseconds-utc-time-in-android))